components/editor/TextEditor.tsx

```diff
@@ -1,15 +1,22 @@
 'use client';
 
+import { DragEventHandler } from 'react';
 import { Editor, EditorContent } from '@tiptap/react';
 import styled from 'styled-components';
 
-const TextEditor = ({ editor }: { editor: Editor | null }) => {
-  return (
+interface editorProps {
+  editor: Editor | null;
+  handleDragOver: DragEventHandler<HTMLDivElement> | undefined;
+  handleDrop: DragEventHandler<HTMLDivElement> | undefined;
```

Comment:
요거 undefined를 조건부로 거는 건 아마 
```suggestion
  handleDragOver?: DragEventHandler<HTMLDivElement>;
  handleDrop?: DragEventHandler<HTMLDivElement>;
```

요렇게는 안 되나요?